src/cliAction/startStopDeleteProject/handler_delete.go

```diff
@@ -0,0 +1,64 @@
+package startStopDeleteProject
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/zerops-io/zcli/src/i18n"
+	"github.com/zerops-io/zcli/src/utils"
+	"github.com/zerops-io/zcli/src/zeropsApiProtocol"
+)
+
+func (h *Handler) RunDelete(ctx context.Context, config RunConfig, projectId string, actionType string) error {
+
+	isConfirmed := string(actionType[0]) == "y"
```

Comment:
you can compare the `rune` here by using single quotes like this `actionType[0] == 'y'`

Reply:
ok